taraxa/state/api.go

```diff
@@ -1,14 +1,15 @@
 package state
 
 import (
+	"math/big"
 	"sort"
 
 	"github.com/Taraxa-project/taraxa-evm/common"
 	"github.com/Taraxa-project/taraxa-evm/core/types"
 	"github.com/Taraxa-project/taraxa-evm/core/vm"
 	"github.com/Taraxa-project/taraxa-evm/rlp"
 	"github.com/Taraxa-project/taraxa-evm/taraxa/state/chain_config"
-	"github.com/Taraxa-project/taraxa-evm/taraxa/state/dpos"
+	dpos "github.com/Taraxa-project/taraxa-evm/taraxa/state/dpos/precompiled"
```

Comment:
same here? "dpos"

Reply:
it is not a typo, that's the name of package, you need to specify it if our files are not in dpos folder but in precompiled